### PR TITLE
Update docs for refactored ISS integration

### PR DESCRIPTION
**Pull request by DurgNomis-drol**
## Proposed change

Removes old configuration explanation.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/64987
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/iss.markdown [hunk 1]
```diff
@@ -18,33 +20,13 @@ This means that ISS is 10° above the horizon of your home.
...
-  type: boolean
-  default: false
-{% endconfiguration %}
+{% include integrations/config_flow.md %}
 
 <div class='note warning'>
 
-If you set `show_on_map: true` then the location attributes are named `latitude` and `longitude`.
-The default name of the location attributes is `lat` and `long` to avoid showing them on the map.
+If you enable show on map when configurating this integration then the location attributes
```
Comment (DurgNomis-drol) [inline source/_integrations/iss.markdown]:
```suggestion
If you enable "Show on map" when configurating this integration then the location attributes 
```

**Review by thecode — DISMISSED** [review]
Thanks @DurgNomis-drol 👍

File: source/_integrations/iss.markdown [hunk 2]
```diff
@@ -18,33 +20,13 @@ This means that ISS is 10° above the horizon of your home.
...
-  type: boolean
-  default: false
-{% endconfiguration %}
+{% include integrations/config_flow.md %}
 
 <div class='note warning'>
 
-If you set `show_on_map: true` then the location attributes are named `latitude` and `longitude`.
-The default name of the location attributes is `lat` and `long` to avoid showing them on the map.
+If you enable "Show on map" when configurating this integration then the location attributes
```
Comment (frenck) [inline source/_integrations/iss.markdown]: This should not be part of the configuration, but part of an option flow, which needs to be described differently.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/iss.markdown (hunk 2 above)
Comment (DurgNomis-drol) [reply source/_integrations/iss.markdown]:
```suggestion
If you enable "Show on map" in the options for this integration then the location attributes 
```

**Review by frenck — APPROVED** [review]
Thanks, @DurgNomis-drol 👍

**Pull request merged**